### ISSUE TITLE
WCAG22 class for span needs a refresh

### ISSUE THREAD

**Issue by bruce-usab** [w3c/wcag#5041]
We have a class, `wcag22`, which (as I understand it’s behavior):
- Prepends “New in WCAG 2.2:” to `span` elements in 2.2 Understanding docs.
- Removes the `span` content from 2.1 Understanding docs.   

It is quite important that we maintain only one set of Understanding documents.  I submit that the first behavior should be improved.
- First, “new” is misleading as WCAG 2.2 is now 2&frac12; years old.  The guidance is no longer “new” in the conventional usage of the word.
- It is helpful to for Understanding content to have some way of noting SC which were added by 2.2.  (This would also be helpful for any 2.0 Understanding which reference SC added by 2.1.  Is there a `wcag21` class?)

I submit that the second behavior, removing content, is harmful (or at the very least, counter-productive).  Users of 2.1 Understanding documents (deliberately, or by accident) are disadvantaged by not being alerted to the fact of the existence of newer material — regardless that 2.2 might not required (or even be consciously of interest to the reader).
- Maybe add &Dagger; to 2.2 SC references in Understanding docs for 2.0 and 2.1 era SC?  There already is a `<a>_sc-short-name_</a>` macro.  That would be sure to catch almost use cases.  (We might also add &dagger; to  to 2.1 SC references in Understanding docs for 2.0 era SC.)  We might also instead use &sup1; and &sup2; for such notation.  And, of course, an explanation of the symbol would be needed on the pages that use it.

FWIW,  #4843 prompted me to create this issues.  Some research is needed to index all uses of the class, so as to inform what (if anything) should be done.  Additional research is desirable to check that there are not more instances which would benefit from the class.

**Comment by kfranqueiro** [comment]
I discussed this with Kevin a couple of weeks ago, and at minimum the class's current functionality is _extremely_ important for [4.1.1 Parsing](https://www.w3.org/WAI/WCAG22/Understanding/parsing), as it is responsible for the content of the first several paragraphs under Intent (through/including the first blockquote), which were specifically written for 2.2.

Every other place it's currently used involves a sentence that links to another SC that only exists in 2.2, which would not make sense to link (relatively) within 2.1 docs:

- [Focus Visible](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible) - a sentence that links to 2.4.13 Focus Appearance
- [Non-text Contrast](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast) - a sentence at the end of Figure 10 that links to 2.4.13 Focus Appearance
- [Pointer Gestures (ED)](https://w3c.github.io/wcag/understanding/pointer-gestures) - two sentences that link to 2.5.7 Dragging Movements

Maybe we can think about how it's presented, but the functionality of filtering out the content for the 2.1 docs seems fairly important.

**Comment by patrickhlauke** [comment]
It may just need a tweak in wording... rather than "*New* in WCAG 2.2:" it could be more timeless, a la "Specific to WCAG 2.2:" or similar

**Comment by bruce-usab** [comment]
Thank you Ken for investigating this.

I submit that rather than “filtering out” content for the 2.1 docs, there should be context _added_ (1) so that people looking specifically for 2.1 have the benefit knowing about improvements with 2.2, and (2) perhaps more importantly, people looking for the most current information get it.  (Maybe these readers aren't attentive to 21 versus 22 in the URL, or maybe it was accidentally to hit 21 instead of 22.)

**Comment by kfranqueiro** [comment]
Kevin and I discussed this a bit today. A few data points and potential alternative thoughts:

- Would it be more natural to present the information naturally with no preamble for anyone reading 2.2, rather than calling out the "New/specific to 2.2" at all?
- Cross-linking from WCAG 2.1 docs to WCAG 2.2 docs (and vice versa) is typically seen as a bug, so that seems like something we would not want to go out of our way to do
- For what it's worth, the Quickref always links to WCAG 2.2, even if you [view the Quickref for 2.1](https://www.w3.org/WAI/WCAG22/quickref/?versions=2.1) (via the WCAG version drop-down in the filter tab), because there is effectively only one version of Quickref
- RE people not realizing which version of the docs they're arriving at, we display the version in the header, but it's not always included in the page title, which might help for search results; that's something I could discuss with Kevin and update the build system to add
